tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -140,30 +142,80 @@ ROOT::Experimental::Detail::RDaosNTupleAnchor::Deserialize(const void *buffer, s
    return result.Unwrap() + 20;
 }
 
-std::uint32_t
-ROOT::Experimental::Detail::RDaosNTupleAnchor::GetSize()
+std::uint32_t ROOT::Experimental::Detail::RDaosNTupleAnchor::GetSize()
 {
-   return RDaosNTupleAnchor().Serialize(nullptr)
-      + ROOT::Experimental::Detail::RDaosObject::ObjClassId::kOCNameMaxLength;
+   return RDaosNTupleAnchor().Serialize(nullptr) +
+          ROOT::Experimental::Detail::RDaosObject::ObjClassId::kOCNameMaxLength;
 }
 
+int FetchNTupleDescriptorBuilder(ROOT::Experimental::Detail::RDaosContainer &cont,
```

Comment:
Consider making this a member function of the `RDaosContainerNTupleLocator` class (maybe `GetNTupleDescriptorBuilder()`?).  There we could initialize the `RNTupleDescriptorBuilder` and return it.